package-lock.json

```diff
@@ -6365,8 +6365,7 @@
     "@popperjs/core": {
```

Comment:
It looks like there's some unwanted changes in here. Can you try reverting the changes to `package-lock.json` and `package.json` and *just* installing sveltejs-tippy?